src/main/java/io/github/thebusybiscuit/slimefun4/utils/BlockUtils.java

```diff
@@ -0,0 +1,32 @@
+package io.github.thebusybiscuit.slimefun4.utils;
+import io.github.thebusybiscuit.cscorelib2.reflection.ReflectionUtils;
+import org.bukkit.Material;
+import org.bukkit.block.Block;
+public final class BlockUtils {
+
+    private static final boolean is_1_14 = ReflectionUtils.isVersion("v1_14_");
```

Comment:
Slimefun has a built-in method for that. And there is no need for making that a field here.

You can do SlimefunPlugin.getMinecraftVersion().isAtLeast(MinecraftVersion.MINECRAFT_1_14) instead.
Your method would break for all 1.15 servers since you check for the exact version, not the minimum.